src/VirtoCommerce.Platform.Web/Startup.cs

```diff
@@ -426,7 +428,16 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 
             app.UseHttpsRedirection();
 
+            // Add default wwwroot static files bindings
             app.UseStaticFiles();
+
+            // Add additional bindings
+            var fileExtensionsBindings = Configuration.GetSection("Content").GetSection("FileExtensionsBindings").Get<Dictionary<string, string>>();
+            app.UseStaticFiles(new StaticFileOptions
+            {
+                ContentTypeProvider = new FileExtensionContentTypeProvider(fileExtensionsBindings)
```

Comment:
Let's add our types to existing `FileExtensionContentTypeProvider`, not to replace it.
Thus we remove all existing builtin type mappings from here:
https://github.com/dotnet/aspnetcore/blob/0e7ca749de460555da0d9c00a351f035d546ec01/src/Middleware/StaticFiles/src/FileExtensionContentTypeProvider.cs